modules/datastore/src/Controller/QueryDownloadController.php

```diff
@@ -126,7 +126,7 @@ private function initStreamedCsvResponse($filename = "data.csv") {
    *   Row of data to send as CSV.
    */
   private function sendRow($handle, array $row) {
-    fputcsv($handle, $row);
+    fputcsv($handle, $row, escape: "\\");
```

Comment:
Goodbye PHP 7.4, but... Do we need to specify the default value for escape?

Also, since we're doing this 8.0+ stuff, we should specify PHP 8.0+ in composer.json.

Reply:
We do, default value is [deprecated](https://www.php.net/manual/en/function.str-getcsv.php). I had to do these manually, actually, and [filed an issue with rector to fix automatically in the future](https://github.com/rectorphp/rector/issues/9057). (I think it might actually be more correct to just use `''` but this seemed more conservative as it was the old default.)

Will make the change in composer.json

Reply:
Nice catch on the deprecation.

Reply:
It was very noisy, hard to miss! Re: version constraint, we don't have one in DKAN now. In our module info we require DKAN ^10.2, and that [requires PHP ^8.1](https://git.drupalcode.org/project/drupal/-/blob/10.2.x/composer.json?ref_type=heads#L58), so I don't think we need to maintain an explicit PHP requirement for our module composer.json?